lib/hooks/api/useRepoStats.ts

```diff
@@ -0,0 +1,19 @@
+import useSWR, { Fetcher } from "swr";
```

Comment:
A new hook to retrieve repository stats. Prwviously, this data was being loaded in `getServerSideProps` which was causing frequent timeouts.